docs/customization.md

```diff
@@ -1,38 +1,55 @@
 ## Customization
-### Modifying the client library format
 
-Don't like the default client syntax?  Want a different language supported?  No problem!  OpenAPI Generator processes mustache templates with the [jmustache](https://github.com/samskivert/jmustache) engine.  You can modify our templates or make your own.
+### Modifying a template
 
-You can look at `modules/openapi-generator/src/main/resources/${your-language}` for examples. To make your own templates, create your own files and use the `-t` flag to specify your template folder.  It actually is that easy.
+Clone OpenAPI Generator and take a look at the following directory: `modules/openapi-generator/src/main/resources/${template}`. In here you'll see all of the generators available, for most programming languages, web application frameworks and web servers. For example, if you are looking for the C# template, it's named `csharp`.
```

Comment:
This still needs to be updated.  You're pointing the user to:

`modules/openapi-generator/src/main/resources/csharp` (where the reader will intuit `${template}` is replaced with `csharp`), and saying that this includes "all of the generators available".

The "all of the generators available" is up a directory, at `modules/openapi-generator/src/main/resources`.

So, either `${template}` would need to be removed, or the description about the directory needs to be modified to accommodate (That is, something like "in here, you'll find all of the templates specific to the generator you plan to modify").

English speakers will read this and go "Oh, that refers to the `resource` directory" but we have many non-English speakers who will be confused by these three introductory sentences.